src/node/entities.h

```diff
@@ -66,8 +66,8 @@ namespace ccf
     // Nodes identities and allowed code ids
     static constexpr auto NODES = "public:ccf.gov.nodes.info";
     static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
-    static constexpr auto NETWORK_CONFIGURATIONS =
-      "public:ccf.gov.network.configurations";
+    static constexpr auto NODES_CONFIGURATIONS =
+      "public:ccf.gov.nodes.network.configurations";
```

Comment:
What does renaming this table mean for backwards compatibility? If there's not a strong reason to rename, I think we should avoid doing so.

Reply:
This is a new table that was added very recently, so no backwards compatibility problems yet. The motivation for renaming is that it now appears after the `nodes` table in lexicographic ordering and we rely on that during the execution of table hooks.